src/components/pages/AuthPage.tsx

```diff
@@ -100,11 +112,28 @@ const AuthPage: FC = () => {
           >
             {t('welcome')}
           </Typography>
-          {isLogin ? (
-            <Login changeComponentType={changeComponentType} />
-          ) : (
-            <Register changeComponentType={changeComponentType} />
-          )}
+          <Box
+            flexGrow="1"
+            display="flex"
+            justifyContent="center"
+            alignItems="center"
+            minHeight="21rem"
+          >
+            {isLoading ? (
```

Comment:
Chaining ternary is usualy a bad practice, so may be would be better use there `&&` and split it into two checks? Like `isLoading && <CircularProgress />`, `!isLoading && ...`